site/content/docs/5.1/components/dropdowns.md

```diff
@@ -457,15 +455,13 @@ Trigger dropdown menus at the left of the elements by adding `.dropstart` to the
 </div>
 
 <!-- Split dropstart button -->
-<div class="btn-group" role="group">
-  <div class="btn-group dropstart" role="group">
-    <button type="button" class="btn btn-secondary dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" aria-expanded="false">
-      <span class="visually-hidden">Toggle Dropstart</span>
-    </button>
-    <ul class="dropdown-menu">
-      <!-- Dropdown menu links -->
-    </ul>
-  </div>
+<div class="btn-group dropstart" role="group">
```

Comment:
[Bootstrap] @louismaximepiton In https://raw.githubusercontent.com/twbs/bootstrap/main/site/content/docs/5.1/components/dropdowns.md `role="group"` has been added only for the dropstart.